strawberry_django/fields/field.py

```diff
@@ -0,0 +1,105 @@
+import strawberry
+from django.db import models
+from strawberry.arguments import UNSET, convert_arguments
+from strawberry.field import StrawberryField
+from .. import utils
+from ..resolvers import django_resolver
+from ..filters import StrawberryDjangoFieldFilters
+from ..ordering import StrawberryDjangoFieldOrdering
+from ..pagination import StrawberryDjangoPagination
+
+
+class StrawberryDjangoFieldBase:
+    def get_queryset(self, queryset, info, **kwargs):
+        type_ = self.type or self.child.type
+        get_queryset = getattr(type_, 'get_queryset', None)
+        if get_queryset:
+            queryset = get_queryset(self, queryset, info, **kwargs)
+        return queryset
+
+    @property
+    def django_model(self):
+        type_ = self.type or self.child.type
+        return utils.get_django_model(type_)
+
+
+class StrawberryDjangoField(
+        StrawberryDjangoFieldOrdering,
+        StrawberryDjangoFieldFilters,
+        StrawberryDjangoPagination,
+        StrawberryDjangoFieldBase,
+        StrawberryField):
+
+    def __init__(self, django_name=None, graphql_name=None, python_name=None, **kwargs):
```

Comment:
Comments explaining the kwargs would be useful, also for the class properties.

Reply:
And docstring to explain the purpose of this class

Reply:
Documentation added